src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+function wrapHeaderIntoSpan(parentElement) {
+  Array.from(parentElement).forEach((child) => {
```

Comment:
Instead of using `Array.from()` for parentElement, consider using` forEach` directly on NodeList, as modern browsers support it.
```suggestion
function wrapHeaderIntoSpan(parentElement) {
  parentElement.forEach((child) => {
```